infra/modules/database/role_manager/role_manager.py

```diff
@@ -0,0 +1,134 @@
+import itertools
+from operator import itemgetter
+import os
+import logging
+from pg8000.native import Connection, identifier
+
+logger = logging.getLogger()
```

Comment:
Usually a name is passed to `getLogger()`, but I'm not sure if that would make a difference here.

Reply:
I think this just returns the root logger, which I thought was fine since I'm trying to keep this lambda simple / to one file.